prometheus/README.md

```diff
@@ -1,55 +1,156 @@
+| auto.d/ProductionDB.yml.example | `/etc/prometheus/auto.d/ProductionDB.yml.example` |
+| crunchy-alert-rules.yml | `/etc/prometheus/crunchy-alert-rules.yml` |
+
+###### Alertmanager
+
+The Alertmanager data directory should be `/var/lib/ccp_monitoring/alertmanager` and owned by the `ccp_monitoring` user.  You can set it up with:
 
-## Start services (RHEL/CENTOS 7)
+```bash
+sudo mkdir /var/lib/ccp_monitoring/alertmanager
+sudo chmod 0700 /var/lib/ccp_monitoring/alertmanager
+sudo chown ccp_monitoring /var/lib/ccp_monitoring/alertmanager
 ```
+
+The following pgmonitor configuration files should be placed according to the following mapping:
+
+| pgmonitor Configuration File | System Location |
+|------------------------------|-----------------|
+| crunchy-alertmanager-service-el7.conf | `/etc/systemd/system/alertmanager.service.d/crunchy-alertmanager-service-el7.conf`  |
+| sysconfig.alertmanager | `/etc/sysconfig/alertmanager` |
+
+
+## Setup
+
+### Setup on RHEL/CentOS 7
+
+#### Service Configuration
+
+The following files contain defaults that should enable Prometheus and Alertmanager to run effectively on your system for the purposes of using pgmonitor.  You should take some time to review them.
+
+If you need to modify them, see the notes in the files for more details and recommendations:
+
+- `/etc/systemd/system/prometheus.service.d/crunchy-prometheus-service-el7.conf`
+- `/etc/systemd/system/alertmanager.service.d/crunchy-alertmanager-service-el7.conf`
+
+The below files contain startup properties for Prometheues and Alertmanager.  Please review and modify these files as you see fit:
+
+- `/etc/sysconfig/prometheus`
+- `/etc/sysconfig/alertmanager`
+
+The below files dictate how Prometheus and Alertmanager will behave at runtime for the purposes of using pgmonitor.  Please review each file below and follow the instructions in order to set things up:
+
+| File                                     | Instructions |
+|------------------------------------------|--------------|
+| `/etc/prometheus/crunchy-prometheus.yml` | Modify to set scrape interval if different from the default of 30s. Activate alert rules and Alertmanager by uncommenting lines when set as needed. Default service expects config file to be named `crunchy-prometheus.yml` |
+| `/etc/prometheus/crunchy-alertmanager.yml` | Setup alert target (e.g., SMTP, SMS, etc.), receiver and route information. Default service expects config file to be named `crunchy-alertmanager.yml` |
+| `/etc/prometheus/crunchy-alert-rules.yml` | Update rules as needed. Default Prometheus config expects file to be named `crunchy-alert-rules.yml` |
+| `/etc/prometheus/auto.d/*.yml` | You will need at least one file with a `.yml` extension in this folder in order to autodiscover and scrape data from your [exporters](exporter/README.md). Copy the example file to create as many additional targets as needed.  Ensure the configuration files you want to use do not end in `.yml.example` but only with `.yml`. |
+
```

Comment:
Last sentence for auto.d instructions seem redundant. Maybe reword the first sentence to say "You will need at least one file with a final `.yml` extension"